dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php

```diff
@@ -140,7 +140,8 @@ public function testCreateCustomerIfInputDataIsEmpty()
 
     /**
      * @expectedException \Exception
-     * @expectedExceptionMessage GraphQL response contains errors: Field CustomerInput.email of required type String! was not provided
+     * @expectedExceptionMessage
+     * GraphQL response contains errors: Field CustomerInput.email of required type String! was not provided
```

Comment:
You can safely remove `GraphQL response contains errors:` part.

Reply:
Done